File: 215-kth-largest-element-in-an-array/215-kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> pqueue = new PriorityQueue<>(nums.length);
        
        for(int val: nums)
            pqueue.add(-val);
        int ans = 0;
        while(k-1 != 0) {
            pqueue.poll();
            k--;
        }
        return -pqueue.peek();
    }
}

File: 215-kth-largest-element-in-an-array/215-kth-largest-element-in-an-array.py
from heapq import *
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        # create a heap
        # pop k times
        inverse = [-1*i for i in nums]
        heapify(inverse)
        num = 0
        while k != 0:
            num = heappop(inverse)
            k -= 1
        return -1 * num
        